deployment/fedora-package-x64/pkg-src/jellyfin.spec

```diff
@@ -77,6 +77,7 @@ EOF
 %{_libdir}/%{name}/*.dll
 %{_libdir}/%{name}/*.so
 %{_libdir}/%{name}/*.a
+%{_libdir}/%{name}/*.xml
```

Comment:
These should not be included IMO. These are documentation files. The build should just not generate these files.

We could include `/p:GenerateDocumentationFile=false;DebugSymbols=false;DebugType=none` as a single command line argument (no spaces). This will generate a very clean build. (Include this command argument at build and at publish time.)